data/series/Battlestar Galactica (2004)/2010-complete-series-blu-ray/disc20.json

```diff
@@ -255,6 +255,12 @@
       "Duration": "0:10:26",
       "Size": 441126912,
       "DisplaySize": "420.6 MB",
+      "Item": {
+        "Title": "My Favorite Episode So Far",
+        "Type": "Extra",
+        "Chapters": [],
+        "Season": "4"
+      },
```

Comment:
Rather than editing the disc json file directly, you should add this information to the disc20-summary.txt. That is the file that determines where extras are located. Then, if you run finalize on the release (using ImportBuddy) it will fill out the correct information in the disc20.json file. If you do not want to finalize the release, please make the necessary additions to the disc20-summary.txt file and I can finalize the json file. Thanks!